assets/js/components/ErrorNotice.js

```diff
@@ -63,7 +64,16 @@ export default function ErrorNotice( {
 		return null;
 	}
 
-	const shouldDisplayRetry = isErrorRetryable( error, selectorData );
+	const shouldDisplayRetry =
+		hasButton && isErrorRetryable( error, selectorData );
+
+	// Append try again messaging if no retry button is present.
+	if ( ! hasButton ) {
+		message = `${ message }. ${ __(
+			'Please try again.',
+			'google-site-kit'
+		) }`;
```

Comment:
We shouldn't be using punctuation inside strings like this where we combine user-facing text if we can help it. This should all be localisable, eg.

```suggestion
		message = sprintf( __(
			'%s. Please try again.',
			'google-site-kit'
		), message );
```

Reply:
Thanks @tofumatt this makes sense, I am going to commit your suggestion.